### ISSUE TITLE
Ubuntu - Errors when "php install.php"

### ISSUE THREAD

**Issue by ve3lst** [myimouto/myimouto#172]
Hi,

I was trying not to create a new issue, but I don't know what to do anymore.

I've been trying to install myimouto for more than a 2 month. First of all using windows 8.1, the installation wasn't a complete success at "php install.php" I face some rails errors i tried 3 time remove every thing and try again but the same errors. The solution to this problem  was assets files you uploaded, and the site started and all but the tags wasn't working.

Then I tried to install myimouto on a fresh Windows 7, got the same problems.

Now that I hate windows so mush, I stopped using it and installed Ubuntu. One of the first things I'm doing is install myimouto, and at the same step "php install.php" I get rails errors, but looks like it's different from the ones I got on Windows 

```
*** PDOException in PDO#__construct ***
could not find driver
#0 Rails/ActiveRecord/Connection.php(268): PDO->__construct('mysql:host=127....', 'root', NULL, Array)
#1 Rails/ActiveRecord/Connection.php(30): Rails\ActiveRecord\Connection->create_connection(Array)
#2 Rails/ActiveRecord/ActiveRecord.php(74): Rails\ActiveRecord\Connection->__construct(Array, 'production')
#3 Rails/ActiveRecord/ActiveRecord.php(60): Rails\ActiveRecord\ActiveRecord::create_connection('production')
#4 Rails/ActiveRecord/ActiveRecord.php(109): Rails\ActiveRecord\ActiveRecord::setConnection('production')
#5 Rails/ActiveRecord/Migration/Migrator.php(15): Rails\ActiveRecord\ActiveRecord::connection()
#6 install.php(13): Rails\ActiveRecord\Migration\Migrator->__construct()
#7 {main}
Error context:
```

Now I'm completely lost, and i don't know what to do any more. Hope you can help me with this issue.

Thank you soooo much for this project :+1:

**Comment by Parziphal** [comment]
According to [here](http://stackoverflow.com/questions/2852748/pdoexception-could-not-find-driver), you are missing PDO MySQL, and according to [here](http://stackoverflow.com/questions/13375061/installing-pdo-driver-on-mysql-linux-server), you can install it by running:

    sudo apt-get install php5-mysql

**Comment by ve3lst** [comment]
Hi,

Thank you for your fast respond and sorry for my late replay. 

I faced some problems installing php5-mysql, anyhow I've done installing it and no more rails errors, thank you so much.

But when I run "php install.php" I get this error

> PHP Parse error:  syntax error, unexpected '[' in /home/USER/Desktop/MyImouto/install.php on line 77

Looks like the problem the old version of php so now I'm installing a newer version, hope that will solve the problem.

**Comment by Parziphal** [comment]
Yes, it seems like you're using PHP 5.3.

**Comment by ve3lst** [comment]
Hi, 

Thank you so much for your help.

The Ubuntu's there was alot of problems that I wasn't able to update PHP (I believe that's because I'm new to linux), any how I ruined  my Ubuntu :V :V :V  and for the next month I'll be busy so I can't fix it at the moment.

However, I've just installed MyImouto on my laptop (Windows 8.1) and looks like it's working perfectly fine. The problem was the port of MySQL  XAMPP (I've MySQL running on 3306 port with password using it for JMM) so I changed the port on XAMPP, and in database.yml and I've got no error and MyImouto worked

Thanks for your help :+1:
